paths.tex

```diff
@@ -331,24 +332,30 @@ \section{ Shortest Path Algorithms}
 of an edge, and have \texttt{edge\_neighborhood} pull it out, giving $O(1)$, but
 requiring precomputation with cost $O(m\log m)$.
 Each algorithm iterates through every vertex, and then in the
-worst case, every edge will enter $R$ at some point. So we have $O(nm\log m)$.
+worst case, every edge will enter $R$ at some point, giving $O(nm \log m)$. Then
+we do this accross all time slices, so we have $O(tnm\log m)$.
 
```

Comment:
Thanks for adjusting this as well. I wish it was as fast as the original `O` times, but timestamps do add just enough complexity to add the `t` multiplier. Do you think it is worth mentioning that for sparse graphs and real-life datasets this `O` time is truly an upper-bound and unlikely to be observed or should that be in a separate document once we run the algs?